### DISCUSSION TITLE
Windows: View the Bluetooth version & supported features of the machine itself

### DISCUSSION THREAD

**Discussion by stamminator** [emericg/toolBLEx#20] [Q&A]
Hello, I just stumbled upon this project while doing some research into Bluetooth LE Audio support on Windows, and I have to say that it's very impressive. But I can't seem to figure out how to accomplish my main objective in installing it: viewing the Bluetooth version, supported features, and any relevant specs of my machine's Bluetooth hardware. I can see the host info prominently displayed on the right, but it's only showing hostname and MAC info. Is there something I'm missing?

![goGYZcG5gx](https://github.com/emericg/toolBLEx/assets/10633170/2244dcf3-a7f5-4e3f-9ea5-07e0cf2eb6be)

**Comment by emericg** [comment]
Hi.

> viewing the Bluetooth version, supported features, and any relevant specs of my machine's Bluetooth hardware

Yes viewing the Bluetooth version for instance is only supported on Linux at the moment (and it's a bit of a hack too).
Properly supporting that feature ("full" host system feature set) is something that I am very interested in, but that can only be done by plugging this application to each of the OS underlying Bluetooth stack.
And that being said, I'm not sure what actual information it would be possible to gather that way, just because I have never seen it in action in another application...

But it's in the TODO list :)
